templates/contribute/naming.md

```diff
@@ -416,3 +415,11 @@ automatically generated unidirectional implications, named `.inj`,
 and there is no intention to change this.
 When such an automatically generated lemma already exists,
 and a bidirectional lemma is needed, it may be named `.inj_iff`.
+
+### Predicates as suffixes
+
+Most predicates should be added as prefixes. Eg `IsClosed (Icc a b)` should be called `isClosed_Icc`, not `Icc_isClosed`.
+
+Some widely used predicates escape this rule. Those are the predicates that are analogous to an atom already suffixed by the naming convention. Here is a non-exhaustive list:
```

Comment:
```suggestion
Some widely used predicates don't follow this rule. Those are the predicates that are analogous to an atom already suffixed by the naming convention. Here is a non-exhaustive list:
```
I would love that this were correct English but unfortunately I don't think that this lovely French idiom can be literally translated.

Reply:
You are chasing the little beast!